# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

kinectActionGoal action_goal
kinectActionResult action_result
kinectActionFeedback action_feedback
